account_index_based_currency/hooks.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+def pre_init_hook(cr):
+    cr.execute(
+        'ALTER TABLE account_invoice ADD COLUMN currency_rate '
```

Comment:
Could you add a docstring to know the rationale of this hook since that the field is defined in columns?